### PR TITLE
Should fix Exemption being shit

### PR DESCRIPTION
**Pull request by JamieD1**
Should fix Exemption being shit

:cl:    
bugfix: Should fix Exemption being shit
/:cl:

### CODE REVIEW

**Comment by KillerOrcaCora** [comment]
Can you dumb it down for me

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -44,8 +44,8 @@
 		body += " played by <b>[M.client]</b> "
 		body += "\[<A href='?_src_=holder;[HrefToken()];editrights=[(GLOB.permissions.admin_datums[M.client.ckey] || GLOB.permissions.deadmins[M.client.ckey]) ? "rank" : "add"];key=[M.key]'>[M.client.holder ? M.client.holder.rank_name() : "Player"]</A>\]"
 		if(CONFIG_GET(flag/use_exp_tracking))
-			body += "\[<A href='?_src_=holder;[HrefToken()];getplaytimewindow=[REF(M)]'>" + M.client.get_exp_living() + "</a> | "
-			body += " <A href='?_src_=holder;[HrefToken()];toggleexempt=[REF(M)]'>Toggle Exempt</a>\]"
+			body += "\[<A href='?_src_=holder;[HrefToken()];getplaytimewindow=[REF(M.client)]'>" + M.client.get_exp_living() + "</a> | "
```
Comment (Bibby0110) [inline code/modules/admin/admin.dm]:
```suggestion
			body += "\[<A href='?_src_=holder;[HrefToken()];getplaytimewindow=[REF(M)]'>" + M.client.get_exp_living() + "</a> | "
```

**Review by Bibby0110 — APPROVED** [review]
